xml/klp.xml

```diff
@@ -294,5 +294,16 @@
     </para>
    </listitem>
   </itemizedlist>
+
+  <sect2 xml:id="sec-klp-lifecycle">
+   <title>Checking expiration date of the live patch</title>
+   <para>
+    Make sure that the
+    <package>lifecycle-data-sle-module-live-patching</package> is installed,
+    then run the <command>zypper lifecycle</command> command. You should see
+    expiration dates for live patches in the <literal>Package end of support if
+    different from product</literal> section of the output.
+   </para>
```

Comment:
Just now I recalled that we have a web page summarizing all the active live patches and their expiration dates: https://www.suse.com/products/live-patching/current-patches/

The suggested addition would be: "At https://www.suse.com/products/live-patching/current-patches/ you can look up the expiration date based on the running kernel version without the need to install the product extension."